test/unit/index.test.js

```diff
@@ -62,4 +77,11 @@ describe('sails-hook-cron', () => {
     assert.equal(secondJob.cronTime.zone, 'Europe/Kiev');
     assert.notOk(secondJob.running);
   });
+
+  it('Should properly load SailsJS', () => {
+    const sailsJob = sails.hooks.cron.jobs.sailsJob;
+
+    assert.isObject(sailsJob);
+    assert.ok(sails.config.sailsWorks);
+  });
```

Comment:
It is a test with side effect, that expects the previous tests results. It should be done in one `it`.